packages/utils/src/lib/execute-process.ts

```diff
@@ -160,9 +161,9 @@ export function executeProcess(cfg: ProcessConfig): Promise<ProcessResult> {
 
     process.on('close', code => {
       const timings = { date, duration: calcDuration(start) };
-      if (code === 0) {
+      if (code === 0 || alwaysResolve) {
         onComplete?.();
-        resolve({ code, stdout, stderr, ...timings });
+        resolve({ code: 0, stdout, stderr, ...timings });
```

Comment:
Keep the original `code`, it could be useful.

```suggestion
        resolve({ code, stdout, stderr, ...timings });
```